modules/pushapk_scriptworker/files/requirements.txt

```diff
@@ -46,7 +46,7 @@ oauth2client==4.1.2
 parso==0.3.0
 pexpect==4.6.0
 pickleshare==0.7.4
-prompt_toolkit==2.0.3
+prompt_toolkit==1.0.15
```

Comment:
You'll need to pin this with the pyup syntax to avoid it being bumped again. Eg: https://github.com/mozilla-releng/build-puppet/blob/master/modules/signingserver/files/linux_requirements.txt#L9